app/src/main/java/com/example/therundown/domain/PlayerMaping.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.therundown.domain
+
+fun PlayerDto.convertToPlayer() = Player(id.toString(), lastName.toString())
```

Comment:
id and last name already strings and we should not use toString for Strings.
You can use Player(id ?: "",  lastName ?: "") or throw exception

Reply:
done